### ISSUE TITLE
Track currently active custom analysis via unique URL

### ISSUE THREAD

**Issue by skept** [neurosynth/neurosynth-web#65]
Currently on the custom analyses page the URL doesn't specify which analysis is currently loaded. Tracking the currently active analysis using some kind of URL fragment would allow the user to easily get back to it and also share it with others.

**Comment by tyarkoni** [comment]
Good idea! Would updating the URL hash with the uuid whenever the load or save buttons are pushed cover it?

**Comment by skept** [comment]
Yes, that covers part of it. The other part is to for the frontend code to parse the URL when it initializes and load the corresponding study. This part gets a little more complicated when the analysis specified by the URL hash is owned by another user.

**Comment by tyarkoni** [comment]
Ah, got it. Along those lines, we should probably talk about implementing permissions of some kind. Right now custom analyses aren't discoverable, so the odds of anyone stumbling onto a valid uuid are pretty much zero. But I would like to eventually allow searching by name/description/included studies/image content, and that will require some kind of privacy setting. But that can be something for the next milestone or two. I'll open a separate issue for it.

**Comment by skept** [comment]
@tyarkoni I've made some good progress on this but it's a bit more complicated than I thought. Once we allow the user to load somebody else's custom analyses into the "analysis manager" interface (via the unique URL) we probably shouldn't show the "Save" and "Delete" buttons, and only allow "Save as". Does that sound reasonable?

**Comment by skept** [comment]
And we should probably rename the "Run" button to "View results" on analyses that are owned by other users.

**Comment by tyarkoni** [comment]
Yep, agreed on both counts!

**Comment by skept** [comment]
@tyarkoni This should be [mostly] working now. I also added a view-only mode to the custom analysis manager interface, which is the default mode when the user loads a previously saved analysis. When you get a chance can you update the test server with the latest commit on the analyses branch?

**Comment by tyarkoni** [comment]
Great! I just updated the test server.

**Comment by skept** [comment]
@tyarkoni See the above commit. I added a button to the analysis results page to link back to the analysis in the custom analysis manager. That way users can make their own copy of the analysis if it looks interesting. I'm still no table to properly run analyses in my dev environment so I haven't been able to actually pull up a rendered version of this page to see if it looks correct. Can you try it out? If it looks good I think we can close this ticket.
